beacon-chain/blockchain/process_block_helpers.go

```diff
@@ -235,7 +236,7 @@ func (s *Service) insertFinalizedDeposits(ctx context.Context, fRoot [32]byte) {
 	// to be included(rather than the last one to be processed). This was most likely
 	// done as the state cannot represent signed integers.
 	finalizedEth1DepIdx := eth1DepositIndex - 1
-	if err = s.cfg.DepositCache.InsertFinalizedDeposits(ctx, int64(finalizedEth1DepIdx)); err != nil {
+	if err = s.cfg.DepositCache.InsertFinalizedDeposits(ctx, int64(finalizedEth1DepIdx), common.Hash(finalizedState.Eth1Data().BlockHash), 0); err != nil {
```

Comment:
This is a requirement for conformance with the spec, for each finalization call we record the execution block height. The reason we put it as 0 for now is because we don't have an easy way currently to provide the execution height. If it helps I can add a comment each time we call this method and use zero.